test/tests.ts

```diff
@@ -43,3 +43,10 @@ test("multi rejects correctly", async t => {
 
     await t.throws(client.execMulti(fakeMulti), "foo");
 });
+
+test("set with expiry", async t => {
+    const client = createHandyClient();
+    await client.set("a:foo", "123", ["EX", 60]);
```

Comment:
@wolever this is how you'll be able to run command-within-a-command style queries.